svir/dialogs/viewer_dock.py

```diff
@@ -79,7 +79,7 @@
 
 
 class ViewerDock(QDockWidget, FORM_CLASS):
-    def __init__(self, iface, action):
+    def __init__(self, iface, action, testing=False):
         """Constructor for the viewer dock.
 
         :param iface: A QGisAppInterface instance we use to access QGIS via.
```

Comment:
testing parameter is not described in the doc string (and the reason why it exists).

Reply:
Done